dist_config.py

```diff
@@ -170,6 +170,44 @@ class _WheelLinuxConfig(TypedDict):
         ],
         'system_packages': '',
     },
+    '13.x': {
+        # CUDA Enhanced Compatibility wheel (for CUDA 12.x)
+        'name': 'cupy-cuda13x',
+        'kind': 'cuda',
+        'platform_version': '13.x',
+        # Use the latest CUDA version for build.
+        # Note: CUDA 12 still supports RHEL/CentOS 7 on x86_64
+        'image': 'cupy/cupy-release-tools:cuda-runfile-13.0.0-centos7',
+        'libs': [],
+        'includes': [],
+        'preloads': ['cutensor', 'nccl', 'cudnn'],
+        'verify_image': 'nvidia/cuda:{system}',
+        'verify_systems': [
+            # Test on all supported CUDA version variants.
+            '13.0.0-runtime-ubuntu22.04',
```

Comment:
Each item in `verify_systems` are expanded to `{system}` in `verify_image`.

We rely on `nvidia/cuda` Docker images for wheel verification but it seems 13.0 variants are not yet available - https://hub.docker.com/r/nvidia/cuda/tags?name=13.0.0